virtcontainers/fc.go

```diff
@@ -737,30 +739,33 @@ func (fc *firecracker) resumeSandbox() error {
 	return nil
 }
 
-func (fc *firecracker) fcAddVsock(vs kataVSOCK) error {
-	span, _ := fc.trace("fcAddVsock")
+func (fc *firecracker) fcAddVsock(hvs types.HybridVSock) error {
+	span, _ := fc.trace("fcAddHybridVsock")
 	defer span.Finish()
 
+	udsPath := hvs.UdsPath
+	if fc.jailed {
+		udsPath = filepath.Join("/", defaultHybridVSocketName)
+	}
+
 	vsockParams := ops.NewPutGuestVsockByIDParams()
 	vsockID := "root"
-	ctxID := int64(vs.contextID)
-	udsPath := ""
+	ctxID := int64(3)
 	vsock := &models.Vsock{
 		GuestCid: &ctxID,
-		UdsPath: &udsPath,
-		VsockID: &vsockID,
+		UdsPath:  &udsPath,
+		VsockID:  &vsockID,
```

Comment:
what am I missing here, just a gofmt change? Is the new format correct?

Reply:
yes, alignment with `GuestCid: &ctxID`